textproc/dblatex/Portfile

```diff
@@ -125,10 +122,9 @@ proc dblatex.mktexlsr {} {
     if {![variant_isset mactex]} {
         texlive.mktexlsr
     } else {
-        global dblatex.mactex_bin
-        system "${dblatex.mactex_bin}/mktexlsr"
-        if [file exists "${dblatex.mactex_bin}/mtxrun"] {
-            system "${dblatex.mactex_bin}/mtxrun --generate"
+        system "mktexlsr"
+        if {[llength [auto_execok mtxrun]]} {
+            system "mtxrun --generate"
```

Comment:
This is not really related to your PR, but I'm pretty sure that running `mtxrun` here makes no sense. It has to be done when new fonts or other useful resources are added to the TeX tree. I see no use of `mtxrun` for `dblatex` though.

Reply:
I've removed the call to `mtxrun`.